subprojects/docs/src/snippets/toolingApi/eclipse/tests/sanityCheck.sample.conf

```diff
@@ -1,2 +1,2 @@
 executable: gradle
-args: tasks
+args: build
```

Comment:
Why is this a sanity check test?

Reply:
Yeah, it should be reverted as we discussed